docs/en/intro/install.md

```diff
@@ -156,174 +169,259 @@ use Cycle\Annotated\Annotation\Column;
+your entities pass, undergoing transformations, validations, and finally, integration into the ORM schema.
+
+```php index.php
 use Cycle\Schema;
 use Cycle\Annotated;
 use Doctrine\Common\Annotations\AnnotationRegistry;
 
-// autoload annotations
+$dbal = /** ... */;
+
+// Initialize annotations
 AnnotationRegistry::registerLoader('class_exists');
 
 $schema = (new Schema\Compiler())->compile(new Schema\Registry($dbal), [
-    new Schema\Generator\ResetTables(),             // re-declared table schemas (remove columns)
-    new Annotated\Embeddings($classLocator),        // register embeddable entities
-    new Annotated\Entities($classLocator),          // register annotated entities
-    new Annotated\TableInheritance(),               // register STI/JTI
-    new Annotated\MergeColumns(),                   // add @Table column declarations
-    new Schema\Generator\GenerateRelations(),       // generate entity relations
-    new Schema\Generator\GenerateModifiers(),       // generate changes from schema modifiers
-    new Schema\Generator\ValidateEntities(),        // make sure all entity schemas are correct
-    new Schema\Generator\RenderTables(),            // declare table schemas
-    new Schema\Generator\RenderRelations(),         // declare relation keys and indexes
-    new Schema\Generator\RenderModifiers(),         // render all schema modifiers
-    new Annotated\MergeIndexes(),                   // add @Table column declarations
-    new Schema\Generator\SyncTables(),              // sync table changes to database
-    new Schema\Generator\GenerateTypecast(),        // typecast non string columns
+    new Schema\Generator\ResetTables(),             // Reconfigure table schemas (deletes columns if necessary)
+    new Annotated\Embeddings($classLocator),        // Recognize embeddable entities
+    new Annotated\Entities($classLocator),          // Identify annotated entities
+    new Annotated\TableInheritance(),               // Setup Single Table or Joined Table Inheritance
+    new Annotated\MergeColumns(),                   // Integrate @Table column annotations
+    new Schema\Generator\GenerateRelations(),       // Define entity relationships
+    new Schema\Generator\GenerateModifiers(),       // Apply schema modifications
+    new Schema\Generator\ValidateEntities(),        // Ensure entity schemas adhere to conventions
+    new Schema\Generator\RenderTables(),            // Create table schemas
+    new Schema\Generator\RenderRelations(),         // Establish keys and indexes for relationships
+    new Schema\Generator\RenderModifiers(),         // Implement schema modifications
+    new Annotated\MergeIndexes(),                   // Merge @Table index annotations
+    new Schema\Generator\SyncTables(),              // Align table changes with the database
+    new Schema\Generator\GenerateTypecast(),        // Typecast non-string columns
```

Comment:
Should we use attributes there instead of annotations?

Reply:
I mean `@Table`, `@Column`

Reply:
+ will fix it